crates/holochain_sqlite/tests/db_wrapper.rs

```diff
@@ -1,16 +1,17 @@
 use common::TestDatabaseKind;
 use holochain_sqlite::conn::set_connection_timeout;
+use holochain_sqlite::db::num_read_threads;
 use holochain_sqlite::db::set_acquire_timeout;
 use holochain_sqlite::db::DbWrite;
-use holochain_sqlite::db::{num_read_threads, PermittedConn, WriteManager};
-use holochain_sqlite::error::DatabaseError;
 use holochain_sqlite::error::DatabaseError::{DbConnectionPoolError, Timeout};
-use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
+use holochain_sqlite::error::{DatabaseError, DatabaseResult};
+use std::sync::atomic::{AtomicUsize, Ordering};
 use std::sync::Arc;
 
 #[cfg(feature = "test_utils")]
 mod common;
 
+// TODO this test can be removed once `conn` is removed from the public interface
```

Comment:
leftover or reserved for future?

Reply:
There are still a few test uses of `conn` that I think are better tackled after the p2p_* modules are moved out of the holochain_sqlite crate